CSS/3-recent-works.css

```diff
@@ -0,0 +1,234 @@
+/* RECENT WORKS */
+
+.recent_works_container {
+    height: auto;
+  }
+  
+  .recent_works_container div {
+    width: 80%;
+    height: 20vh;
+    margin: auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  
+  .recent_works_container div h1 {
+    color: #091e42;
+    font-family: "Crete Round", serif;
+    width: fit-content;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+  
+  .recent_works_container div .line {
+    height: 1px;
+    width: 65%;
+    background-color: #212121;
+  }
+  
+  .recent_works_container figure {
+    display: flex;
+    height: 80vh;
+    width: 80%;
+    margin: auto;
+  }
+  
+  .recent_works_container figure img {
+    width: 648px;
+    height: 448px;
+  }
+  
+  .recent_works_container figcaption {
+    height: 448px;
+    padding: 0 2rem;
+    display: flex;
+    gap: 1rem;
+    flex-direction: column;
+    justify-content: center;
+  }
+  
+  .recent_works_container figcaption h1 {
+    font-family: "Crete Round", serif;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+  
+  .recent_works_container figcaption p {
+    font-family: "Inter", sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+  
+  .recent_works_container figcaption ul {
+    display: flex;
+    gap: 1rem;
+  }
+  
```

Comment:
- [ ] AS mentioned by the previous reviewer some of your feature are not matching the given design. kindly use [grid](https://www.w3schools.com/css/css_grid.asp) and [flex box ](https://www.w3schools.com/css/css3_flexbox.asp)to fix this.

here is your design
![Screenshot from 2022-05-28 23-52-39](https://user-images.githubusercontent.com/65860678/170844043-cebcd543-9e3d-4adf-aef0-7c6f71da5087.png)

Reply:
I have already completed all the requirements, so what are your requirements? please clarify